GOVERNANCE.md

```diff
@@ -0,0 +1,189 @@
+* A no-confidence vote is triggered when a core team member (i.e Council member or Core contributor) calls for one publicly on an appropriate project communication channel, and two other core team members second the proposal. The initial call for a no-confidence vote must specify which type is intended -- whether it is targeting a single member or the council as a whole.
+* The vote lasts for two weeks, and the people taking part in it vary:
+* If this is a single-member vote called by Core contributors, both Council members and Core contributors vote, and the vote is deemed successful if at least two thirds of voters express a lack of confidence.
+* If this is a whole-council vote, then it was necessarily called by Core contributors (since Council members can’t remove the whole Council) and only Core contributors vote. The vote is deemed successful if at least two thirds of voters express a lack of confidence.
+* If this is a single-member vote called by Council Members, only Council Members vote, and the vote is deemed successful if at least half the voters express a lack of confidence. Council Members also have the possibility to call for the whole core team to vote (i.e Council members and Core contributors), although this is not the default option. The threshold for successful vote is also at 50% of voters for this option.
+* If a single-member vote succeeds, then that member is removed from the council and the resulting vacancy can be handled in the usual way. If a whole-council vote succeeds, the council is dissolved and a new council election is triggered immediately.
+
+#### Ejecting core contributors
+* Core contributors can be ejected through a simple majority vote.
+* Upon ejecting a core contributor the council must publish an issue ticket, or public document detailing the
+  * Violations
+  * Evidence if available
+  * Remediation plan (if necessary)
+  * Signatures majority of council members to validate correctness and accuracy
+* Core contributors can also voluntarily leave the project by notifying the community through a public means or by notifying the entire council.
+
+#### One time Decisions
+* The first election will also include a vote for the number of Council Members, options being 5 and 7. A simple majority will indicate preference.
+  * At the conclusion of the vote this section should be removed
+
+
+#### Voting criteria for future elections
+Voting for first election is restricted to establish stable governance, and to defer major decision to elected leaders
+* For the first election only the folks in Slack can vote (excluding GSOC students)
+* In the first year, the council must determine voting eligibility for future elections between two criteria:
+  * Those with commit bits
```

Comment:
I'd say core developers instead of people wit commit bits to stick to the bodies defined above

Reply:
Its not defined. Its one of the actions the council needs to take once we elect them

Reply:
Yeah that's pretty wide, but maybe it's ok for the first election? IIUC, refining the definition will be on the first Council's to-do lis.
In any case, it's indeed closer to "Core contributor" than "Core Dev" -- related to our discussion above.

Reply:
Edited my comment to say core contributors. The main point is to stick with the bodies defined in the community architecture section where we define steering council, core contributors and general contributors

Reply:
Everything should say developers now. Leaving this comment open unti I'm sure I didnt mess up

Reply:
Yes............. (of course I mess up in the comment)